examples/sigstore-go-signing/main.go

```diff
@@ -34,12 +34,15 @@ var idToken *string
 var intoto *bool
 var tsa *bool
 var rekor *bool
+var signingconfigJSONpath *string
 
 func init() {
 	idToken = flag.String("id-token", "", "OIDC token to send to Fulcio")
 	intoto = flag.Bool("in-toto", false, "Content to sign is in-toto document")
 	tsa = flag.Bool("tsa", false, "Include signed timestamp from timestamp authority")
 	rekor = flag.Bool("rekor", false, "Including transparency log entry from Rekor")
+	signingconfigJSONpath = flag.String("signingconfigJSONpath", "", "Path to signingconfig JSON file")
```

Comment:
I'd have a shorter flag name and use `-` rather than camel case, maybe `--signing-config-path`?